src/main/java/io/jenkins/plugins/analysis/core/model/SourcePrinter.java

```diff
@@ -129,6 +131,20 @@ private ContainerTag createTitleAndCollapsedDescription(final String message, fi
     private UnescapedText unescape(final String message) {
         return new UnescapedText(SANITIZER.render(message));
     }
+    
+    private ArrayList<DomContent> replaceNewLine(final String message) {
+    	ArrayList<DomContent> messageLines = new ArrayList<DomContent>();
+    	String[] messageDivided = message.split("\n");
+    	if(messageDivided.length > 1) {
+    		for(int i = 0; i < messageDivided.length; i++) {
+        		messageLines.add(pre(unescape(messageDivided[i])));
+        	}
+    	}else {
+    		messageLines.add(unescape(messageDivided[0]));
+    	}
+    	
+    	return messageLines;
+    }
```

Comment:
Please reformat with my coding style settings (in IntelliJ you should already have the correct formatter installed).